### PR TITLE
V2

### PR DESCRIPTION
**Pull request by envin3**
This branch will become the default one. multichain will be removed.

### CODE REVIEW

**Review by oliviera9 — APPROVED** [review]
LGTM

File: src/components/App.jsx [hunk 1]
```diff
@@ -43,23 +43,15 @@ const router = createHashRouter([
 const settings = getWeb3Settings()
 
 const { chains, provider } = configureChains(
-  [mainnet, polygon, bsc],
+  [polygon],
```
Comment (oliviera9) [inline src/components/App.jsx:46]: I am not completely sure it's a good idea to remove other chains from here as the provider for mainnet is used to fetch proposal for example.

Comment (envin3) [reply src/components/App.jsx:46]: This was requested.

**Comment by oliviera9** [comment]
Closing this PR as v2 is not the default branch

**Pull request closed without merging**